src/SonarScanner.MSBuild.Shim/SonarScanner.Wrapper.cs

```diff
@@ -99,7 +99,7 @@ private static bool InternalExecute(AnalysisConfig config, IEnumerable<string> u
             var scannerCliFolder = Path.Combine(scannerCliDirectoryLocation, $"sonar-scanner-{scannerVersion}");
 
             // During packaging of the artifacts (see script https://github.com/SonarSource/sonar-scanner-msbuild/blob/master/scripts/package-artifacts.ps1)
-            // the scanner-cli is unzipped only for the case of .NET Framework 4.6.
+            // the scanner-cli is unzipped for .NET Framework 4.6, but not for the .NET and .NET Core - where it's unzipped upon first usage of the scanner.
```

Comment:
I would not put the 4.6 there, as the .net framework version will change soon and the behavior will be the same.

Reply:
```suggestion
            // the scanner-cli is unzipped for .NET Framework, but not for the .NET and .NET Core - where it's unzipped upon first usage of the scanner.
```